src/common/SmartUrlInput.js

```diff
@@ -97,13 +99,19 @@ export default class SmartUrlInput extends PureComponent<Props, State> {
     onChangeText(fixRealmUrl(autocompleteRealm(value, { protocol, domain })));
   };
 
+  _keyboardDidHide = () => {
+    const { textInputRef } = this;
+    if (textInputRef) {
+      textInputRef.current.blur();
```

Comment:
As of now , the keyboard doesn't hide when the user presses on <code>textinput</code> , instead it maintains its previous state . (As shown in GIF) .
Would love to change according to what makes more sense .